src/main/kotlin/app/revanced/patches/shared/settings/preference/impl/SwitchPreference.kt

```diff
@@ -16,15 +13,21 @@ import org.w3c.dom.Element
  * @param default The default value of the switch.
  * @param summaryOn The summary to show when the preference is enabled.
  * @param summaryOff The summary to show when the preference is disabled.
+ * @param userDialogMessage The message to show in a dialog when the user toggles the preference.
  */
 internal class SwitchPreference(
     key: String, title: StringResource,
     val default: Boolean = false,
     val summaryOn: StringResource? = null,
-    val summaryOff: StringResource? = null
+    val summaryOff: StringResource? = null,
+    userDialogMessage: StringResource? = null
 ) : BasePreference(key, title) {
     override val tag: String = "SwitchPreference"
 
+    init {
+        userDialogMessage?.include()
+    }
```

Comment:
While this technically works, it isn't really a clean solution. Instead, the string should be merged the same way as the other parameter strings in the `serialize` method below.